e2e/ia.conf.js

```diff
@@ -8,6 +8,8 @@ module.exports = {
   UseProxy: process.env.TEST_E2E_USE_PROXY !== 'false',
   WaitForAngular: process.env.TEST_E2E_WAIT_FOR_ANGULAR !== 'false',
   FailFast: process.env.TEST_E2E_FAIL_FAST !== 'false',
+  NightlyTag: process.env.TEST_E2E_ANNOTATION || '--cucumberOpts.tags=@nightly-test',
```

Comment:
Check this `||` condition is required as you are setting the TEST_E2E_ANNOTATION